troubleshooting for repl timeout (#3834)
circleci/circleci-docs: jekyll/_cci2/proxy.adoc

=== Troubleshooting

==== Can't access the Management Console
If you cannot access the CircleCI Management Console, but the Services machine seems to be running, try to SSH tunnel into the machine by running the following, substituting your proxy address and the IP address of your Services machine:

```shell
ssh -L 8800:<address you want to proxy through>:8800 ubuntu@<ip_of_services_machine>
```

> ==== Repl times out

Author: ```suggestion
==== REPL time out
```

> If you experience a timeout when running the following, you will need to allow access through your corporate proxy to any repositories that are required for dependency management:

Author: Yep that's clearer! Will change...

> If you experience a timeout when running the following, you will need to allow access through your corporate proxy to any repositories that are required for dependency management:

Author: ```suggestion
If you experience a timeout when connecting to the REPL, you will need to allow access, through your corporate proxy, to the domains of any Clojure library repositories that are required to download dependencies for running the REPL:
```

> If you experience a timeout when connecting to the REPL, you will need to allow access, through your corporate proxy, to the domains of any Clojure library repositories that are required to download dependencies for running the REPL:

Author: ```suggestion
If you experience a timeout when connecting to the REPL, you will need to allow access, through your corporate proxy, to the domains of any Clojure library repositories that are required to download dependencies for running the REPL.
```
